code/modules/clothing/spacesuits/rig/vault_hardsuit_token.dm

```diff
@@ -0,0 +1,34 @@
+/obj/item/vault_token
```

Comment:
Remove this. Makes no sense for this to exist in the vault, since the safes are not something that only antags will interact with.